pkg/telemetrylogs/field_mapper.go

```diff
@@ -106,23 +123,41 @@ func (m *fieldMapper) FieldFor(ctx context.Context, key *telemetrytypes.Telemetr
 		return "", err
 	}
 
+	if !querybuilder.BodyJSONQueryEnabled && key.FieldContext == telemetrytypes.FieldContextBody {
+		return "", errors.NewInvalidInputf(errors.CodeInvalidInput, "Group by/Aggregation isn't available for the body column")
+	}
```

Comment:
The `FieldFor` shouldn't return this error. It should only be returned in the context of grouping / aggregation.